app/js/services/filter.service.js

```diff
@@ -619,6 +629,11 @@ function FilterService(
             });
         }
 
+        //age
+        if (includeParam('age') && _age !== 0) {
```

Comment:
this should be 
`if (includeParam('age') && _age !== '0') {`

the age is being stored as a string

Reply:
Nice catch! Thanks Josh.